requirements_test_brain.txt

```diff
@@ -3,7 +3,10 @@ types-attrs
 nose
 numpy>=1.17.0; python_version<"3.11"
 python-dateutil
+PyQt5>=5.15.2
```

Comment:
I think we can all-in with the brain test requirements and add major libraries. We have very expansive primer for pylint, brain test are comparatively very cheap for astroid. We probably need to define what constitute a major library and what should be maintained in an astroid-plugin though.